pattern/lib.h

```diff
@@ -183,6 +183,8 @@ int mutt_which_case(const char *s);
 int mutt_is_list_recipient(bool all_addr, struct Envelope *e);
 int mutt_is_subscribed_list_recipient(bool all_addr, struct Envelope *e);
 int mutt_pattern_func(int op, char *prompt);
+int mutt_pattern_alias_func(int op, char *prompt, struct AliasMenuArray *marray,
+                            struct Menu *m);
```

Comment:
Build warnings from using `struct AliasMenuArray` without declaring it.
Add, this after the `#include`s
```c
struct AliasMenuArray;
```